src/python/pants_release/release.py

```diff
@@ -1108,8 +1000,6 @@ def create_parser() -> argparse.ArgumentParser:
     subparsers.add_parser("test-release")
     subparsers.add_parser("build-wheels")
     subparsers.add_parser("build-fs-util")
-    subparsers.add_parser("build-local-pex")
-    subparsers.add_parser("build-universal-pex")
```

Comment:
> It isn't universal, but we're moving away from the universal PEX.

So I understand, this PR is doing two things: the very nifty addition of uploading platform-specific PEXes, and also hard-removing the universal PEX. A few questions:

1. Has this decision to no longer support a universal PEX been confirmed? The only discussion I've seen is https://pantsbuild.slack.com/archives/C0D7TNJHL/p1685645383249239?thread_ts=1685644290.953559&cid=C0D7TNJHL which didn't really come to a conclusion, but maybe it's been discussed elsewhere?
2. Is this change for future releases on `main` only (e.g. 2.18.0.dev5, etc), or will it apply to older releases (2.16.1, 2.17.0) etc?
3. Is there an existing plan for communicating the deprecation/removal?
4. If there's 'controversy'/discussion that needs to happen around this, maybe this clean-up/removal could happen separately, to avoid blocking the platform-specific PEX improvement behind deciding about the universal PEXes?

Reply:
1. Yeah we talked about it in the meeting, although we should also publicize it. Also this from @benjyw https://github.com/pantsbuild/pants/pull/19028#issuecomment-1552275043
2. `main` only, I think.
3. I suspect #general is as good as it gets
4. meh?

FWIW the universal PEX isn't well known why we still publish it (except because we were, and therefore we are). As I understand it was something used at some point for _reasons_, but likely isn't the case today.

Furthermore, a universal PEX can still be created if absolutely necessary (instead of consuming the per-platform PEXs) from the individual PEXs.

I can handle the announcement.

Reply:
Cool, sounds good 👍 I agree 4 isn't necessary, since the decision seems to be confirmed.

For 3, maybe we could also add a note like "this is the last release that publishes the universal PEX, if you're using it, let us know what for!" or something to #19168. Up to you.

Reply:
I commented on that PR :+1: